.github/workflows/build.yml

```diff
@@ -36,12 +32,54 @@ jobs:
             echo "date=$(date +%Y%m%d)" >> $GITHUB_OUTPUT
             alias_tags+=("latest")
           fi
+
           echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
 
+  push-ghcr:
+    name: Build and push image
+    runs-on: ${{ matrix.build_arch == 'aarch64' && 'ubuntu-24.04-arm' || 'ubuntu-24.04' }}
+    needs:
+      - calculate-tags
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        build_arch: [aarch64, x86_64]
+    steps:
+      # Checkout push-to-registry action GitHub repository
+      - name: Checkout Push to Registry action
+        uses: actions/checkout@v4
+
+      # Update to tags to add the arch to the tags
+      - name: Update tags
+        id: updated-tags
+        run: |
+          tags=(${{ needs.calculate-tags.outputs.alias_tags }})
+          updated_tags=()
+          for tag in "${tags[@]}"; do
+            updated_tags+=("${tag}-${{ matrix.build_arch }}")
+          done
+          echo "alias_tags=${updated_tags[*]}" >> $GITHUB_OUTPUT
+
+          echo "sha_short=${{ needs.calculate-tags.outputs.sha_short }}-${{ matrix.build_arch }}" >> $GITHUB_OUTPUT
+
+          # If date is not empty string, append the arch to the date
+          if [[ -n "${{ needs.calculate-tags.outputs.date }}" ]]; then
+            echo "date=${{ needs.calculate-tags.outputs.date }}-${{ matrix.build_arch }}" >> $GITHUB_OUTPUT
+          fi
+
       - name: Check just syntax
         id: check_just_syntax
         uses: ublue-os/just-action@v2
 
+      - name: Install skopeo & podman
```

Comment:
arm runners are nerfed a little, ran by a third party which isn't github (weird situation tbh!).

I've just opted to explicitly install both of these deps regardless of arch